tools/remove_jiv_data_from_db_and_filesystem.pl

```diff
@@ -0,0 +1,153 @@
+#! /usr/bin/perl
+
+
+=pod
+
+=head1 NAME
+
+remove_jiv_data_from_db_and_filesystem.pl -- Cleans up the JIV data from the
+database tables and the filesystem
+
+=head1 SYNOPSIS
+
+perl remove_jiv_data_from_db_and_filesystem.pl C<[options]>
+
+Available option is:
+
+-profile: name of the config file in ../dicom-archive/.loris_mri
+
+=head1 DESCRIPTION
+
+This script will remove the JIV files from the C<parameter_file> table and
+move them to the C<$data_dir/archive/bkp_jiv_produced_before_LORIS_20.0> directory of the filesystem for
+projects that wish to clean up the JIV data produced in the past. Note that
+from release 20.0, JIV datasets will not be produced anymore by the imaging
+insertion scripts.
+
+=head2 Methods
+
+=cut
+
+use strict;
+use warnings;
+
+use Getopt::Tabular;
+use File::Copy;
+
+use NeuroDB::DBI;
+use NeuroDB::ExitCodes;
+
+
+
+my $profile;
+my $profile_desc = "name of config file in ../dicom-archive/.loris_mri";
+
+my @opt_table = (
+    [ "-profile", "string", 1, \$profile,
+        "name of config file in ../dicom-archive/.loris_mri"
+    ]
+);
+
+my $Help = <<HELP;
+
+This script will remove entries in the parameter_file table for the JIV files
+ and backup the JIV directory in /data/project/data to the archive folder.
+
+Documentation: perldoc remove_jiv_data_from_db_and_filesystem.pl
+
+HELP
+
+my $Usage = <<USAGE;
+
+Usage: $0 -help to list options
+
+USAGE
+
+&Getopt::Tabular::SetHelp($Help, $Usage);
+&Getopt::Tabular::GetOptions(\@opt_table, \@ARGV) || exit 1;
+
+
+
+
+## Input option error checking
+if ( !$profile ) {
+    print $Help;
+    print STDERR "$Usage\n\tERROR: missing -profile argument\n\n";
+    exit $NeuroDB::ExitCodes::PROFILE_FAILURE;
+}
+{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
+if ( !@Settings::db ) {
+    print STDERR "\n\tERROR: You don't have a \@db setting in the file "
+        . "$ENV{LORIS_CONFIG}/.loris_mri/$profile \n\n";
+    exit $NeuroDB::ExitCodes::DB_SETTINGS_FAILURE;
+}
+
+
+
+
+## establish database connection
+my $dbh  = &NeuroDB::DBI::connect_to_db(@Settings::db);
+print "\n==> Successfully connected to database \n";
+
+
+
+
+## select the JIV parameter type ID from parameter_type
+my $query = "SELECT ParameterTypeID FROM parameter_type WHERE Name='jiv_path'";
+my $sth   = $dbh->prepare($query);
+$sth->execute();
+
+my $row = $sth->fetchrow_hashref();
+if ( !$row ) {
+    print "\n==> Did not find any entry in the parameter_type table with "
+          . "Name='jiv_path'. Exiting now.\n";
+    exit $NeuroDB::ExitCodes::SUCCESS;
```

Comment:
I think we are existing with success prematurely here;
I found this out during testing as I ran this before, so no more entries in the table, but the files are still there, so I never get to backing up the files with this exit here...
but in real situation, we would have both I assume, the jiv/ directory and the entries in parameter_file table, so if this is run one time, then all should be fine, but re-running might cause issues...